shell/platform/darwin/ios/framework/Source/SemanticsObjectTest.mm

```diff
@@ -644,6 +644,18 @@ - (void)testFlutterSwitchSemanticsObjectMatchesUISwitch {
   XCTAssertEqual(object.accessibilityValue, nativeSwitch.accessibilityValue);
 }
 
+- (void)testSemanticsObjectDeallocated {
+  fml::WeakPtrFactory<flutter::AccessibilityBridgeIos> factory(
+      new flutter::MockAccessibilityBridge());
+  fml::WeakPtr<flutter::AccessibilityBridgeIos> bridge = factory.GetWeakPtr();
+  SemanticsObject* parent = [[SemanticsObject alloc] initWithBridge:bridge uid:0];
+  SemanticsObject* child = [[SemanticsObject alloc] initWithBridge:bridge uid:1];
+  parent.children = @[ child ];
+  __weak SemanticsObject* weakObject = parent;
+  parent = nil;
+  XCTAssertNil(weakObject);
```

Comment:
The probably could use a comment that it's making sure dealloc doesn't crash, it looks like it's not testing anything other than that `__weak` is working.